test/Microsoft.SqlTools.ServiceLayer.UnitTests/QueryExecution/DataStorage/SaveAsExcelFileStreamWriterHelperTests/[Content_Types].xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not entirely certain, but do these files need to be committed? Or are they leftovers from running the tests?

If the latter, you can utilize the MemoryFileSystem in Tests.Common to store any generated files in memory rather than storing them on the disk and cleaning up afterwards.